<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KARMA" ignored="2" name="Default test(failed)" tests="1" failures="1" timestamp="2020-12-28T18:47:57 CST" time="0.341" errors="0">
  <testcase name="test2" time="0.095" classname="org.testng.rerunningfailedtests.RerunFailedTestsFromResults">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="org.testng.rerunningfailedtests.RerunFailedTestsFromResults">
    <ignored/>
  </testcase> <!-- test1 -->
  <testcase name="test3" time="0.0" classname="org.testng.rerunningfailedtests.RerunFailedTestsFromResults">
    <ignored/>
  </testcase> <!-- test3 -->
</testsuite> <!-- Default test(failed) -->
